src/Mimi.jl

```diff
@@ -225,6 +228,7 @@ function addcomponent(m::Model, t, name::Symbol;before=nothing,after=nothing)
     end
 
     ComponentReference(m, name)
+    add_node(m.model_graph, name)
 end
```

Comment:
Yes, I am going to make the ComponentInstanceInfo type that we discussed, so that nodes will end up holding all of the component's information (right now they are just symbols)